### PR TITLE
[TASK] Small readme updates

### PR DESCRIPTION
**Pull request by erikhansen**
Added links to a couple of helpful resources regarding email client support for CSS

Added instructions for installing PHPUnit and running tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +146,9 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, [install PHPUnit](https://phpunit.de/getting-started.html) and then run this
```
Comment (oliverklee) [inline README.md]: Actually, you should install the development-related composer packages using composer:

composer install

(omitting the --no-dev option then also installs the development packages).

PHPUnit will then be available as vendor/bin/phpunit .

File: README.md [hunk 2]
```diff
@@ -143,6 +146,9 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
...
+      command in the root of the GIT repo: 
+      `phpunit --bootstrap Classes/Emogrifier.php  Tests/Unit/EmogrifierTest.php`
```
Comment (oliverklee) [inline README.md]: We also don't need the bootstrap parameter here.

**Comment by oliverklee** [comment]
I've fixed the PHP_CodeSniffer warnings on Travis. Could you please rebase and re-push? Thanks!

**Comment by erikhansen** [comment]
@oliverklee - I've rebased the commit and updated the document to cover the proper method of installing phpunit. Thanks for informing me of that. I guess this experience demonstrates the value of having such instructions in the readme.

File: README.md [hunk 3]
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
+     
+      `composer install` 
+     
+      and then run this command:
+      
+      `vendor/bin/phpunit Tests/Unit/EmogrifierTest.php`
```
Comment (oliverklee) [inline README.md]: Actually, providing just the Tests/ directory as a parameter should suffice.

File: README.md [hunk 4]
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
```
Comment (oliverklee) [inline README.md]: Could you please adapt the indentation to match the indentation of the previous lines?

File: README.md [hunk 5]
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
...
+     
+      `composer install`
```
Comment (oliverklee) [inline README.md]: For code lines, if you indent with four spaces, you won't need the backticks for marking the text as code.

Please see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code for a Markdown cheat sheet.

File: README.md [hunk 6]
```diff
@@ -143,6 +146,18 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
...
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, run the following command:
+     
+      `composer install` 
+     
+      and then run this command:
+      
+      `vendor/bin/phpunit Tests/Unit/EmogrifierTest.php`
+      
+      If you want to uninstall the development packages that were installed when you ran `composer install`, you can
```
Comment (oliverklee) [inline README.md]: Could you please change the (new) Markdown lines to be 80 characters or less? This will avoid wrapping on GitHub.

**Comment by oliverklee** [comment]
For the commit message:

* Could you please wrap the lines in the description at 72 characters?
* Could you please a bullet lists (using asterisks) for the lines in the description that are no paragraphs?

For more information on good commit messages, please see http://chris.beams.io/posts/git-commit/ .

**Comment by erikhansen** [comment]
@oliverklee I've made the requested changes and pushed the rebased commit.

**Comment by oliverklee** [comment]
Thanks!

**Pull request merged**